online_test.go

```diff
@@ -400,15 +400,29 @@ func TestProxyProto(t *testing.T) {
 	}
 }
 
-func TestHostname(t *testing.T) {
+var letters = []byte("abcdefghijklmnopqrstuvwxyz")
+
+func randString(n int) string {
+	b := make([]byte, n)
+	for i := range b {
+		b[i] = letters[rand.Intn(len(letters))]
+	}
+	return string(b)
+}
+
+func TestDomain(t *testing.T) {
 	paidTest(t)
 	ctx := context.Background()
 
+	prefix := randString(8)
+	domain := fmt.Sprintf("%s.ngrok.io", prefix)
+	expectedUrl := fmt.Sprintf("https://%s", domain)
+
 	tun, exited := serveHTTP(ctx, t, nil,
-		config.HTTPEndpoint(config.WithDomain("foo.robsonchase.com")),
+		config.HTTPEndpoint(config.WithDomain(domain)),
```

Comment:
Does this change in any ways `TestSubdomain`? Cause it looks kind of the same? Also `ngrok.io`?

Reply:
*Huh*, they are the same! :sweat_smile: 

Cool, just gonna axe the one I was messing with then.